src/components/Chat.js

```diff
@@ -756,50 +760,59 @@ const Chat = (props) => {
     <Flex
       w="100%"
       h="100vh"
+      paddingTop={10}
       justify="center"
       align="center"
       className="chat-component">
+
       <Flex w="100%" h="90%" flexDir="column">
-        <div id="conversations">
-          {convoList &&
+
+        <HStack
+        className='mobileHStackChat'
+         w="80%"
+
+         borderTop={'40px'}
+         backgroundSize='50px'
+         align="center"
+         justify="center"
+         style={{margin: 10}}
+
+         >
+            <Select
+              variant='filled'
+              s='lg'
+              w='md'
+              align='center'
+
+              placeholder='Contact List'
+              onChange={(evt) => {
+              setList([])
+              setInterlocutor(evt.target.value)
+            } }>
+            {convoList &&
             convoList.map((conversation, idx) => {
-              if (interlocutor && interlocutor === conversation.id) {
-                return (
-                  <Button
+              return (
+
+                  <option
+                  text
                     key={idx + conversation.id}
-                    style={{ margin: 10 }}
-                    bg="lightgrey"
-                    border="2px solid black"
-                    onClick={() => {
-                      setList([]);
-                      setInterlocutor(conversation.id);
-                    }}>
+                    value={conversation.id}
+                    >
                     {conversation.name}
-                  </Button>
-                );
-              }
-              return (
-                <Button
-                  key={idx + conversation.id}
-                  style={{ margin: 10 }}
-                  bg="lightgrey"
-                  onClick={() => {
-                    setList([]);
-                    setInterlocutor(conversation.id);
-                  }}>
-                  {conversation.name}
-                </Button>
-              );
-            })}
+                  </option>
+
+
+             )})}
+             </Select>
           {!account ? (
             <Button
               onClick={connectWallet}
-              style={{ margin: 10, marginTop: 30 }}>
+              style={{}}>
```

Comment:
if there aren't styles could we remove the whole thing? like delete the line with 'style={{}}'